test/02LessonTwoTest.js

```diff
@@ -8,11 +8,24 @@ function whichSpecies(character) {
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
 
+    if (character === 'scooby') {
+      return "dog";
+    } else if (character === 'garfield') {
+      return "cat";
+    } else if (character === 'nemo') {
+      return "fish";
+    } else {
+      return false;
+    }
 }
 
 function isEven(number) {
     // should return true is number is even (divisible by 2)
-
+    if (number % 2 === 0) {
+      return true;
+    } else {
+      return false;
+    }
 }
 
```

Comment:
@jasonburks Great work! Really appreciate that you're indenting the code properly!